ci/local/build.sh

```diff
@@ -76,7 +76,7 @@ mkdir -p "${REPO_PATH}/${PYTHON_BUILD_DIR}"
 
 BUILD_SCRIPT="#!/bin/bash
 set -e
-WORKSPACE=${REPO_PATH_IN_CONTAINER}
+export WORKSPACE=${REPO_PATH_IN_CONTAINER}
```

Comment:
Was this not working before?

Reply:
not working before adding export. invalid path errors during notebook testing because workspace wasn't set.